### PR TITLE
Update publication documentation to the single krill binary setup

### PR DESCRIPTION
**Pull request by timbru**

### CODE REVIEW

File: source/publication-server.rst [hunk 1]
```diff
@@ -54,7 +63,7 @@ following settings:
 .. code-block:: bash
```
Comment (AlexanderBand) [inline source/publication-server.rst:63]: I'd use `.. code-block:: text` here.

**Review by AlexanderBand — COMMENTED** [review]
In the intro, maybe add a paragraph on what function the Publication Server provides. Bascially answering the question: The CA generates some cryptographic objects which nees to be be served by an RRDP and rsync server. So what does the pub server do? Also, how does this relate to the repository server. If this is explained elsewhere, maybe cross reference it.

File: source/publication-server.rst [hunk 2]
```diff
@@ -35,14 +35,23 @@ local Publication Server:
...
+In principle you can enable the Publication Server on the same Krill instance
+that you use to operate your CAs. But, it may be better to use a separate
+instance for this purpose. This will allow more fine grained access control
+to either instance, and it makes it somewhat easier to parse the log files
+in case of issues.
 
+At some point we were more opinionated about this separation and we almost
+split up the Krill binary into two separate binaries for the CA and Publication
+Server functions. However, in the end we decided that you should be able to make
+your own judgement call.
```
[the remark this answers is not in the document]
Comment (timbru) [reply source/publication-server.rst:52]: ack

File: source/publication-server.rst (hunk 1 above)
Comment (timbru) [reply source/publication-server.rst:63]: ack

**Comment by timbru** [comment]
The contextual overview is pretty clear if you look at the rendered page, although it's somewhat hard to read in the diff I suppose: https://krill.docs.nlnetlabs.nl/en/0.9.0/publication-server.html

**Pull request merged**